tensorflow/lite/kernels/test_util.h

```diff
@@ -65,6 +65,12 @@ inline std::vector<float> Dequantize(const std::vector<T>& data, float scale,
   return f;
 }
 
+float GetQuantizeTolerance(int min, int max) {
+  float QuantizedStep = (max - min) / 255.0;
```

Comment:
nit: Name QuantizedStep quantized_step and same for other names.

Reply:
Yes, will update!